dotcom-rendering/src/web/components/TopicFilterBank.importable.tsx

```diff
@@ -84,7 +84,9 @@ export const TopicFilterBank = ({
 }: Props) => {
 	const palette = decidePalette(format);
 	const selectedTopic = selectedTopics?.[0];
-	const topFiveTopics = availableTopics.slice(0, 5);
+	const topFiveTopics = availableTopics
+		.slice(0, 5)
+		.filter((topic) => !!topic.count && topic.count > 2);
```

Comment:
Do you want to `filter` first and then `slice`? Otherwise you might have miss topics with more count that are outside the first 5.

Event if there is an assumption that the topics are sorted by count from the backend, that’s quite a loose contract, so it’s probably worth updating this.

Reply:
yea good idea. I'll make another PR to change the order of filter and slice 👍